jest.config.js

```diff
@@ -9,16 +12,17 @@ module.exports = {
   moduleNameMapper: {
     '\\.(s?css)$': 'identity-obj-proxy',
     '@openmrs/esm-framework': '@openmrs/esm-framework/mock',
-    '^lodash-es/(.*)$': 'lodash/$1',
-    '^uuid$': '<rootDir>/node_modules/uuid/dist/index.js',
     '^dexie$': require.resolve('dexie'),
+    '^lodash-es/(.*)$': 'lodash/$1',
+    '^react-i18next$': path.resolve(__dirname, 'react-i18next.js'),
+    '^uuid$': path.resolve(__dirname, 'node_modules', 'uuid', 'dist', 'index.js'),
```

Comment:
What does path.resolve do?

Reply:
Resolves the path relative to the current working directory. As discussed on the call, we shouldn't need this, and I've reverted back to using `<rootDir>`.